src/PhpParser/NodeFinder/LocalMethodCallFinder.php

```diff
@@ -44,16 +44,16 @@ public function match(ClassMethod $classMethod): array
         $matchingMethodCalls = [];
 
         foreach ($methodCalls as $methodCall) {
-            $callerType = $this->nodeTypeResolver->getType($methodCall->var);
-            if (! $callerType instanceof TypeWithClassName) {
+            if (! $this->nodeNameResolver->isName($methodCall->name, $classMethodName)) {
```

Comment:
doing the cheaper check first so we don't need to resolve the type of every `methodcall->var`